Controllers/PredictionController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using covid19_api.Common.Constants;
+using covid19_api.Models;
+using covid19_api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace covid19_api.Controllers
+{
+    [Route(ApiConstants.DefaultController)]
+    public class PredictionController : ControllerFacade
+    {
+        private readonly ICovidService _covidService;
+
+        public PredictionController(ICovidService covidService)
+        {
+            _covidService = covidService;
+        }
+
+        /// <summary>
+        /// Gets the specified country.
+        /// </summary>
+        /// <param name="country">The country.</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route(ApiConstants.CountryParam)]
+        public Task<IEnumerable<CovidPrediction>> Get([FromRoute] string country)
```

Comment:
let's keep all the endpoints asynchronous

Reply:
okey... this method is returning a task , the endpoint callers will make it async by calling the method , like in a Promise in js, or did not I catch the async comment? or maybe was another function?

Reply:
nvm this is cool this way